tools/ci/install-annex-snapshot.sh

```diff
@@ -10,7 +10,7 @@ if [ -e "$_ANNEXDIR" ]; then
 fi
 echo "I: downloading and extracting under $_ANNEXDIR"
 tar -C $(dirname $_ANNEXDIR) -xzf <(
-  wget -q -O- https://downloads.kitenet.net/git-annex/linux/current/git-annex-standalone-amd64.tar.gz
+  wget -q -O- https://downloads.kitenet.net/git-annex/autobuild/amd64/git-annex-standalone-amd64.tar.gz
```

Comment:
so `current` is the released, and `autobuild` is the snapshot?  I guess need to add the autobuild to #4640 , heh.

Reply:
Yeah, AFAICT autobuild is more or less the most recent commit (of course with some amount of lag).